master/pkg/tasks/task.go

```diff
@@ -83,6 +83,28 @@ func ToContainerSpec(t TaskSpec) container.Spec {
 		shmSize = t.TaskContainerDefaults.ShmSizeBytes
 	}
 
+	var capAdd []string
+	if env.AddCapabilities != nil {
+		capAdd = *env.AddCapabilities
+	}
+
+	var capDrop []string
+	if env.DropCapabilities != nil {
+		capDrop = *env.DropCapabilities
+	}
+
+	resources := t.ResourcesConfig()
+	var devices []docker.DeviceMapping
+	if resources.Devices != nil {
+		for _, device := range *resources.Devices {
```

Comment:
i think just this will work fine unless i'm missing something
```suggestion
	for _, device := range resources.Devices {
```

Reply:
It will, but only after removing the pointer from resources.Devices, I believe.